package.json

```diff
@@ -4,7 +4,7 @@
   "license": "UNLICENSED",
   "scripts": {
     "build": "vite build",
-    "dev": "cross-env NODE_ENV=development node vite-server.js",
+    "dev": "vite",
```

Comment:
If we can get these changes working we can remove the cross-env and express dependencies.